qiskit/circuit/qpy_serialization.py

```diff
@@ -842,12 +926,16 @@ def _write_instruction(file_obj, instruction_tuple, custom_instructions, index_m
         )
         or gate_class_name == "Gate"
         or gate_class_name == "Instruction"
-        or isinstance(instruction_tuple[0], (library.BlueprintCircuit, library.PauliEvolutionGate))
+        or isinstance(instruction_tuple[0], (library.BlueprintCircuit))
```

Comment:
strange brackets here
```suggestion
        or isinstance(instruction_tuple[0], library.BlueprintCircuit)
```